packages/web/src/scss/components/Dropdown/README.md

```diff
@@ -1,30 +1,97 @@
 # Dropdown
 
 ```html
-<div class="Dropdown is-open">
-  <a href="#" class="d-flex mb-400">
-    <svg width="24" height="24" aria-hidden="true" class="mr-400">
-      <use xlink:href="/icons/svg/sprite.svg#info" />
-    </svg>
-    <span>Information</span>
-  </a>
-  </a>
-  <a href="#" class="d-flex mb-400">
-    <svg width="24" height="24" aria-hidden="true" class="mr-400">
-      <use xlink:href="/icons/svg/sprite.svg#link" />
-    </svg>
-    <span>More links</span>
-  </a>
-  <a href="#" class="d-flex mb-400">
-    <svg width="24" height="24" aria-hidden="true" class="mr-400">
-      <use xlink:href="/icons/svg/sprite.svg#profile" />
-    </svg>
-    <span>Profile</span>
-  </a>
-  <a href="#" class="d-flex">
-    <svg width="24" height="24" aria-hidden="true" class="mr-400">
-      <use xlink:href="/icons/svg/sprite.svg#help" />
-    </svg>
-    <span>Help</span>
+<div class="dropdown">
+  <button
+    data-toggle="dropdown"
+    data-target="DropdownDemo2"
+    data-reference="parent"
+    data-placement="top-start"
+    data-offset="5"
+    data-padding="5"
+  >
+    toggle
+  </button>
+  <div class="Dropdown is-open" id="DropdownDemo2">
```

Comment:
Again, for better accessibility, the dropdown should have a label. It can utilize either `aria-label` or `aria-labelledby` linked to the ID of the trigger button.